### DISCUSSION TITLE
Send suricata alerts

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#4354] [Unsupported Versions]
Hi,
Im trying to send Suricata alerts from the new Security Onion 2.3 to an external Arcsight connector (over tls).
Unlike the old version of security onion, this version does not have syslog-ng.
Is there a different way to send the alerts to an external connector?

**Comment by weslambert** [comment]
Please see the following:

https://docs.securityonion.net/en/latest/logstash.html#forwarding-events-to-an-external-destination

**Reply by yula-21** [reply]
Thank you for the reply,
I was able to send alerts in http but now I want to send over https.
I have read the following: https://www.elastic.co/blog/configuring-ssl-tls-and-https-to-secure-elasticsearch-kibana-beats-and-logstash

The problem is that I need to send logs to an external arcsight server and I dont know if the steps are the same.
Another problem is that all of the configuration files locations are different in SO 2.3
For example: /etc/elasticsearch does not exist.
Do you have any suggestions?
